test/tests/benchmark/runtime_tests.py

```diff
@@ -0,0 +1,118 @@
+from config.benchmark_config import *
+from config.amqp import *
+from modules.logger import Log
+from modules.amqp import AMQPWorker
+from on_http import NodesApi as Nodes
```

Comment:
lib name `on_http` updated. You need to generate the python client code with the latest on-http code